.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "tabWidth": 2,
```

Comment:
This file looks good, i like no semis, my personal preference is for tabWidth to be 4 but again at the end of the day it doesnt really matter

Reply:
My preference is for 4 as well